core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -70,6 +79,13 @@ var PreferencesPage = function() {
     correctOptions.first().click();
   };
 
+  this.setUserBio = function(bio) {
+    userBioElement.clear();
+    userBioElement.sendKeys(bio);
+    navBar.click();
```

Comment:
Can you please add a comment above why we are doing this?

Also, it will get saved just by clicking on the `navBar`, if yes then you can remove clicking p[referencesLink.